libafl_targets/src/cmplog.h

```diff
@@ -152,6 +155,7 @@ static inline void cmplog_instructions_extended_checked(
   hits &= CMPLOG_MAP_H - 1;
   libafl_cmplog_map_extended_ptr->vals.operands[k][hits].v0 = arg1;
   libafl_cmplog_map_extended_ptr->vals.operands[k][hits].v1 = arg2;
+  libafl_cmplog_map_extended_ptr->vals.operands[k][hits].v0_is_const = 0;
```

Comment:
this "extended" one is only used for llvm pass instrumentation (which is not using sancov stuff so you don't add it for this

Reply:
line 222, too

Reply:
Yeah, hence it is set to `0` for now (unless you wanted to create a separate `CmpLog` struct without the `v0_is_const` member). I think that for longer comparisons (such as u64 or memcmps), the chance of conincidentally matching a const value is lower (apart from values like `0`); so it's probably not worth the extra effort.

Reply:
what i mean is `struct CmpLogInstructionExtended` doesn't have v0_is_const member

Reply:
True